recipes/pydmtools/meta.yaml

```diff
@@ -1,22 +1,25 @@
 {% set name = "pydmtools" %}
-{% set version = "0.1.1" %}
+{% set version = "0.1.6" %}
 
 package:
   name: "{{ name|lower }}"
-  version: {{ version }}
-
-source:
-  url: https://3dgenomics.hzau.edu.cn:10068/tddownload/pydmtools-0.1.1.tar.gz
-  sha256: 5f24ea5d41431509adeed1c834c4efd88c5bb01ec969fb00b30d5f6841de83a9
+  version: "{{ version }}"
 
+source: 
+  url: https://files.pythonhosted.org/packages/9a/86/d4e7ab710a261785d9f4b67c33d314c087e8126397b5cb0b7a05a7947cb1/pydmtools-0.1.6.tar.gz
```

Comment:
it would be great to use a URL here without the checksum and with a templated version number